### DISCUSSION TITLE
Expected speed up over HuggingFace

### DISCUSSION THREAD

**Discussion by deepakdalakoti** [vllm-project/vllm#662] [Q&A]
Hi,

As per the vllm homepage the results show a huge speed up over huggingface API. However, my benchmarking results show only a modest improvement over Huggingface (~15% ). I am using `Llama-2-7b-hf ` for testing.  I used the code and prompt described [here](https://hamel.dev/notes/llm/03_inference.html#vllm) for testing. Is anyone seeing similar results? Are there some settings which should improve the results significantly? I am using the following libraries

CUDA: 11.8
vllm: 0.1.3

**Comment by deelipvenkat** [comment]
same here , not seeing much improvement in latency , although batch inferencing is much faster with vllm compared huggingface.
